docs/UserGuide.md

```diff
@@ -2,1012 +2,1171 @@
+### Finding a course: `find course`
+You can filter your course list to show only courses that contain specific keywords.
+
+**Command Format:**
 
-Format:
 ```
-select <INDEX>
+find course <KEYWORD> [<MORE_KEYWORDS>]
 ```
-- Selects course at index specified in `<INDEX>` and displays list of groups from that course.
 
-Examples:
-- `select 1` will display a list of groups from the first course.
+<div markdown="block" class="alert alert-info">:information_source: **Note:**<br>
 
-[Back to list of commands](#commands)
+* This command is case-insensitive. (e.g. `find course CS2103T` and `find course cs2103t` will give you the same result)
+* Keywords are separated by spaces.
+</div>
+<br>
 
-### Finding a course: `find course`
+Example:
+  If your course list contains 2 courses "CS2103T" and "CS2101":
+  - `find course CS` will show "CS2103T", "CS2101".
+  - `find course 03` will show "CS2103T".
+
+[Back to list of features](#features)
+
+### Selecting a course: `select`
+This command allows you to select a course and navigates you to its corresponding `Group Page`.
 
-Finds course(s) that contains any of the given keywords(s) in the module code.
+**Command Format:**
 
-Format:
 ```
-find course <KEYWORD> [<MORE_KEYWORDS>]
+select <INDEX>
 ```
 
-- This command is case-insensitive. (e.g. `find course CS2103T` and `find course cs2103t` will give you the same result)
-- Keywords are separated by spaces.
-
 Examples:
-- `find course CS` returns `CS2103T`, `CS2101`.
-- `find course 03` returns `CS2103T`.
 
-[Back to list of commands](#commands)
+- `select 1` will display a list of groups from the first course.
+
+[Back to list of features](#features)
+
+---
 
---------------------------------------------------------------------------------------------------------------
+## Managing Groups
+In the following sections, you will find the features that are available to you on the `Group Page`.
+A `Group Page` typically looks like this:
 
-## Group Page Commands
-![image](./images/UiGroupPage.png)
-###### Figure 5
+<img src="./images/UiGroupPage.png" width="650" height="529"/>
+<br>
 
 ### Adding a group: `add group`
 
-Adds a group to the list of groups in the group page.
+Use this command to a new group to a course you've selected.
+
+**Command Format:**
 
-Format:
 ```
 add group <GROUP>
 ```
 
-- Adding a group with the same group name is not allowed.
-    - For example, if a group with group name `T15` already exists in CLIpboard,
-  entering `add group T15` will display an error message inside the result display.
-
-<div markdown="block" class="alert alert-info">
-This command is case-sensitive.<br>
-e.g. <code>T15</code> and <code>t15</code> are treated as different groups.
+<div markdown="block" class="alert alert-info">:information_source: **Note:**<br>
+  * Adding groups with the same group name is not possible.
+  * Group names can contain alphanumeric and special characters. e.g. "AY2223S2-CS2103T"
```

Comment:
```suggestion
  * Group names can contain alphanumeric and special characters. e.g. "L03-2"
```
Would be better to change the module to group names here.

Reply:
thanks fixed